awx_collection/tests/integration/targets/project/tasks/main.yml

```diff
@@ -53,6 +53,23 @@
         that:
           - result is not changed
 
+    - name: Create a git project and wait with short request timeout.
+      project:
+        name: "{{ project_name1 }}"
+        organization: Default
+        scm_type: git
+        scm_url: https://github.com/ansible/test-playbooks
+        wait: true
+        state: exists
+        request_timeout: .1
```

Comment:
I'd like to think that at some point this will become flaky because we can process this in under 100ms, if it can support even shorter timeouts I would prefer that.

Reply:
Updated to .001 and tested that to make sure it kept the error message, changed from auth token to handshake timeout.